app/src/main/java/pomobox/data/model/BreakTask.java

```diff
@@ -0,0 +1,71 @@
+package pomobox.data.model;
+
+import java.io.Serializable;
+
+public class BreakTask implements Serializable {
+    private int mTaskId;
+    private int mIsDone;
+    private String mTaskTitle;
+    private String mTaskContent;
+    private int mTargetPomo;
+    private int mProgressPomo;
+
+    public BreakTask() {
+    }
+
+    public BreakTask(int isDone, String taskTitle, String taskContent, int progressPomo, int targetPomo) {
```

Comment:
vs nhưng constructor mà cần từ  5 params trở lên a nên sử dụng Builder Pattern
Ref: https://dzone.com/articles/design-patterns-the-builder-pattern